packages/vaadin-template-renderer/package.json

```diff
@@ -27,7 +27,7 @@
     "@esm-bundle/chai": "^4.3.4",
     "@vaadin/checkbox": "^22.0.0-alpha6",
     "@vaadin/testing-helpers": "^0.3.0",
-    "@vaadin/vaadin-grid": "^22.0.0-alpha6",
+    "@vaadin/grid": "^22.0.0-alpha6",
```

Comment:
nit: let's sort dependencies alphabetically (so that grid would be after checkbox).